File: commons/src/main/scala/com/oceanum/common/TaskStatus.scala
package com.oceanum.common

object TaskStatus extends Enumeration {
  type value = Value
  val OFFLINE: value = Value(0)
  val PREPARE: value = Value(1)
  val START: value = Value(2)
  val RUNNING: value = Value(3)
  val FAILED: value = Value(4)
  val SUCCESS: value = Value(5)
  val RETRY: value = Value(6)
  val TIMEOUT: value = Value(7)
  val KILL: value = Value(8)
}

File: commons/src/main/java/com/oceanum/common/TaskStatus.java
package com.oceanum.common;

/**
 * @author chenmingkun
 * @date 2020/8/9
 */
public enum  TaskStatus {

    OFFLINE(0),
    PREPARE(1),
    START(2),
    RUNNING(3),
    FAILED(4),
    SUCCESS(5),
    RETRY(6),
    TIMEOUT(7),
    KILL(8);

    int value;

    TaskStatus(int value) {
        this.value = value;
    }
}
